### PR TITLE
feat: Article table component

### PR DESCRIPTION
**Pull request by muhsin-k**
Fixes [#5017 ](https://github.com/chatwoot/product/issues/472)
**Preview**
Screenshot 2022-07-12 at 4 46 39 PM

### CODE REVIEW

File: app/javascript/dashboard/components/helpCenter/ArticleTable.vue [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    text-transform: capitalize;
+    color: var(--s-600);
+    font-size: var(--font-size-small);
+    padding-left: 0;
+  }
+  .horizontal-line {
+    border-bottom: 1px solid var(--color-border);
+  }
+  .footer {
+    padding: 0px;
```
Comment (iamsivin) [inline app/javascript/dashboard/components/helpCenter/ArticleTable.vue:81]:
```suggestion
    padding: 0;
```

**Review by iamsivin — DISMISSED** [review]
LGTM 👍🏻

**Review by fayazara — DISMISSED** [review]
Lgtm

File: app/javascript/dashboard/components/helpCenter/ArticleTable.vue [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+};
+</script>
+<style lang="scss" scoped>
+.article-container {
+  width: 100%;
+
+  table thead th {
+    font-weight: var(--font-weight-bold);
+    text-transform: capitalize;
+    color: var(--s-600);
```
Comment (nithindavid) [inline app/javascript/dashboard/components/helpCenter/ArticleTable.vue:73]:
```suggestion
    color: var(--s-700);
```

**Review by nithindavid — CHANGES_REQUESTED** [review]

**Review by nithindavid — APPROVED** [review]

**Pull request merged**